docs/modules/commons-operator/pages/reference/environment-variables.adoc

```diff
@@ -0,0 +1,34 @@
+= Environment variables
+
+This operator accepts the following environment variables:
+
+== KUBERNETES_CLUSTER_DOMAIN
+
+*Default value*: cluster.local
+
+*Required*: false
+
+*Multiple values*: false
+
+This instructs the operator, which value it should use for the Kubernetes `clusterDomain` setting.
+Make sure to keep this in sync with whatever setting your cluster uses.
+Please see the documentation xref:guides:kubernetes-cluster-domain.adoc[on configuring the Kubernetes cluster domain] for more information on this feature.
+
+[source]
+----
+export KUBERNETES_CLUSTER_DOMAIN=mycluster.local
+cargo run -- run
+----
+
+or via docker:
+
+[source]
+----
+docker run \
+--name commons-operator \
+--network host \
+--env KUBECONFIG=/home/stackable/.kube/config \
+--env KUBERNETES_CLUSTER_DOMAIN=mycluster.local \
+--mount type=bind,source="$HOME/.kube/config",target="/home/stackable/.kube/config" \
+docker.stackable.tech/stackable/commons-operator:0.0.0-dev
```

Comment:
There is no "latest" tag. `0.0.0-dev` works but not sure how this works with different doc and release versions. @xeniape are you already that deep into the docs to have an opinion :)?

Reply:
The stand-alone Docker examples should be removed and no new ones should be added. They are misleading, incomplete and really just a relic of the olden _agent-outside-of-kubernetes_ days.

Reply:
Yeah thought about that as well, will remove.